pkg/manager/member/ticdc_upgrader.go

```diff
@@ -44,6 +44,25 @@ func (u *ticdcUpgrader) Upgrade(tc *v1alpha1.TidbCluster, oldSet *apps.StatefulS
 	ns := tc.GetNamespace()
 	tcName := tc.GetName()
 
+	if tc.Status.PD.Phase == v1alpha1.UpgradePhase ||
+		tc.Status.TiKV.Phase == v1alpha1.UpgradePhase ||
+		tc.Status.TiFlash.Phase == v1alpha1.UpgradePhase ||
+		tc.Status.Pump.Phase == v1alpha1.UpgradePhase ||
+		tc.Status.TiDB.Phase == v1alpha1.UpgradePhase {
+		klog.Infof("TidbCluster: [%s/%s]'s pd status is %s, "+
```

Comment:
Why couldn't the Phase of TiCDC become `ScalePhase`?

Reply:
It's just because no need to care about the scale phase during upgrading, but for PD, TiKV, TiFlash we'd better avoid scaling and upgrading at the same time because it may cause unexpected behavior.